topomodelx/nn/combinatorial/hmc_layer.py

```diff
@@ -0,0 +1,1075 @@
+"""Higher-Order Attentional NN Layer for Mesh Classification."""
+
+
+from typing import List
```

Comment:
Please don't use `typing.List` anymore, it's deprecated. Use the built-in `list`.